src/components/attendee/InputField/InputField.module.scss

```diff
@@ -0,0 +1,21 @@
+@use "scss/attendee/space";
+
+.inputFieldWrapper {
+  position: relative;
+}
```

Comment:
do you use anywhere relative position of the wrapper? looks like you're using flex now, so I guess it's not needed?

Reply:
yep, you're right
removed, [8cad6f9](https://github.com/sparkletown/sparkle/pull/2948/commits/8cad6f9d0156b47d6c22bf2b03bbf3ceac0e2c40)